### PR TITLE
doc comment rewording

### PR DESCRIPTION
**Pull request by king6cong**

### CODE REVIEW

File: src/librustc/mir/mod.rs [hunk 1]
```diff
@@ -63,7 +63,7 @@ macro_rules! newtype_index {
 }
 
 /// Lowered representation of a single function.
-// Do not implement clone for Mir, its easy to do so accidently and its kind of expensive.
+/// Do not implement clone for Mir, which can be accidently done and kind of expensive.
```
Comment (frewsxcv) [inline src/librustc/mir/mod.rs]: Looks good!

Regarding your `//` to `///` change, I'm not sure if this information belongs in the doc comment. What do you think?

Comment (king6cong) [inline src/librustc/mir/mod.rs]: Yes, better to keep it out of doc comment. Thanks

**Comment by king6cong** [comment]
@frewsxcv Updated

**Comment by frewsxcv** [comment]
Looks good, thanks!

@bors r+ rollup

**Pull request merged**